automation/test.sh

```diff
@@ -432,7 +432,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   fi
 fi
 
-if [[ $KUBEVIRT_NONROOT =~ true ]]; then
+if [[ $FEATURE_GATES != *"Root"* ]]; then
```

Comment:
nit: I know that `NonRoot` is the default behavior, so it is not mandatory to set the FG.
But this chek will skip non root tests if someone will use 
`export FEATURE_GATES=NonRoot`

It's probably not worth considering.

Reply:
Saying that, good catch on the pattern matching ;)